README.md

```diff
@@ -8,14 +8,15 @@
 
 `ndg` is a way to automatically generate an HTML document containing all the options you use in any set of modules.
 
- This flake exposes a package (`packages.<system>.ndg-builder` a.k.a. `packages.<system>.default`),
- as well as an overlay (`overlays.default`) to allow accessing `ndg-builder` in another nixpkgs instance.
+This flake exposes a package (`packages.<system>.ndg-builder` a.k.a. `packages.<system>.default`),
+as well as an overlay (`overlays.default`) to allow accessing `ndg-builder` in another nixpkgs instance.
 
 ## Usage
 
 You can override the exposed package with the following options:
 
-* `rawModules`: a list of modules containing `options` to document. For example:
+- `rawModules`: a list of modules containing `options` to document. For example:
```

Comment:
any reason to prefer `-` over `*`?

Reply:
Not really, it's just something my formatter does